dataflux_pytorch/dataflux_mapstyle_dataset.py

```diff
@@ -140,23 +150,26 @@ def __getitem__(self, idx):
                 bucket_name=self.bucket_name,
                 object_name=self.objects[idx][0],
                 retry_config=self.config.download_retry_config,
-            ))
+            )
+        )
 
     def __getitems__(self, indices):
         return [
-            self.data_format_fn(bytes_content) for bytes_content in
-            dataflux_core.download.dataflux_download_threaded(
+            self.data_format_fn(bytes_content)
+            for bytes_content in dataflux_core.download.dataflux_download_threaded(
                 project_name=self.project_name,
                 bucket_name=self.bucket_name,
                 objects=[self.objects[idx] for idx in indices],
                 storage_client=self.storage_client,
-                dataflux_download_optimization_params=self.
-                dataflux_download_optimization_params,
+                dataflux_download_optimization_params=self.dataflux_download_optimization_params,
                 threads=self.config.threads_per_process,
                 retry_config=self.config.download_retry_config,
             )
         ]
 
+    def data_format_default(self, data):
```

Comment:
Even if we do need to have this as a separate function, i don't see any reason for it to be a member of the class. Can we define it outside of the class scope?

Reply:
Done